test/general.js

```diff
@@ -34,4 +50,26 @@ describe('general', () => {
       fac._stop(done)
     }
   })
+
```

Comment:
can we follow examples with Employees instead of current SELECT queries so we keep same standard on tests

Reply:
sure sure